docs/application/dotnet/guides/connectivity/webview.md

```diff
@@ -0,0 +1,165 @@
+
+    ```
+    webView.LoadUrl("https://www.tizen.org/");
+    ```
+
+- The following table lists the loading events provided by the `Tizen.WebView.WebView` class:
+
+**Table: Loading Events**
+
+| Event            | Description                                            | Argument                                 |
+|------------------|--------------------------------------------------------|------------------------------------------|
+| `LoadStarted`    | This event occurs when loading starts.                 | -                                        |
+| `LoadFinished`   | This event occurs when loading finishes.               | -                                        |
+| `LoadError`      | This event occurs when an error appears while loading. | Tizen.WebView.SmartCallbackLoadErrorArgs |
+
+- Add an event handler to handle each loading event, as shown in the following code snippet:
+
+    ```
+    webView.LoadStarted += (s, e) =>
+    {
+        /* Handle LoadStarted event here */
+    };
+
+    webView.LoadFinished += (s, e) =>
+    {
+        /* Handle LoadFinished event here */
+    };
+
+    webView.LoadError += (s, e) =>
+    {
+        /* Handle LoadError event here */
+
+        // error code
+        var code = e.Code;
+
+        // description
+        var desc = e.Description;
+
+        // URL
+        var url = e.Url;
+    };
+    ```
+
+<a name='cookie'></a>
+## Managing Cookie
+
+To manage and set the cookie options, use the [Tizen.WebView.CookieManager](https://samsung.github.io/TizenFX/latest/api/Tizen.WebView.CookieManager.html) class.
+
+- To get the `Tizen.WebView.CookieManager` object from WebView, use `Tizen.WebView.WebView.GetContext` and `Tizen.WebView.Context.GetCookieManager`, as shown in the following code snippet:
+
+    ```
+    CookieManager cookieManager = webView.GetContext().GetCookieManager();
+    ```
+
+- To set the cookie acceptance policy, use the `Tizen.WebView.CookieManager.SetCookieAcceptPolicy` method, as shown in the following code snippet:
+
+    ```
+    /* Accepts every cookie sent from any page */
+    cookieManager.SetCookieAcceptPolicy(CookieAcceptPolicy.Always);
+
+    /* Rejects all the cookies */
+    cookieManager.SetCookieAcceptPolicy(CookieAcceptPolicy.Never);
+
+    /* Accepts only cookies set by the main document that is loaded */
+    cookieManager.SetCookieAcceptPolicy(CookieAcceptPolicy.NoThirdParty);
+    ```
+
+- To set the persistent storage for the cookie, use the `Tizen.WebView.CookieManager.SetPersistentStorage` method, as shown in the following code snippet:
+
+    ```
+    cookieManager.SetPersistentStorage(DirectoryInfo.Data, CookiePersistentStorage.SqlLite);
+    ```
+
+<a name="finalize"></a>
+## Finalizing WebView
+To clean up the resources allocated by your application, call the `Tizen.WebView.Chromium.Shutdown` method after the end of the main loop, as shown in the following code snippet:
```

Comment:
@younghajung 
Kindly change it to: 
To clean up the allocated resources, call the `Tizen.WebView.Chromium.Shutdown` method after the end of the main loop, as shown in the following code snippet:

Reply:
Done